sklearn/utils/weight_vector.pyx.tp

```diff
@@ -1,3 +1,27 @@
+{{py:
+
+"""
+Efficient (dense) parameter vector implementation for linear models. 
+
+Template file for easily generate fused types consistent code using Tempita
+(https://github.com/cython/cython/blob/master/Cython/Tempita/_tempita.py).
+
+Generated file: weight_vector.pxd
+
+Each class is duplicated for all dtypes (float and double). The keywords
+between double braces are substituted in setup.py.
+"""
+
+# name, c_type
+dtypes = [('64', 'double'),
+          ('32', 'float')]
+
+def get_dispatch(dtypes):
```

Comment:
Sorry not familiar with tempita (and the tempita doc links are broken): why do you need get_dispatch? You can't simply iterate over the `dtypes` list?